CI/travis.linux.before_install.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -ev
+
+exit 0
```

Comment:
This was a quick hack to test whether `sudo`-less installation works for trusty at all. The files will disappear in the final version of this PR. I was just lazy here.